os/board/artik05x/src/artik05x_symtab.c

```diff
@@ -0,0 +1,104 @@
+/*****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <stdint.h>
+#include <stdio.h>
+#include <assert.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <arch/board/symtab.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+#define MAX_SYMBOL_NAME_LEN 256
+/****************************************************************************
+ * Private Variables
+ ****************************************************************************/
+static uint32_t symtab_base_addr = 0;
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+int up_symtab_flashdrv_init(void)
+{
+	char *a;
+	char *b;
+	int index = 0;
```

Comment:
I feel coding guidelines variable naming convention can be followed for better code readability
ex:
index_for_a
index_for_b

Reply:
okay. Will modify accordingly.